.github/workflows/main.yml

```diff
@@ -89,9 +89,11 @@ jobs:
           cd extensions/ql-vscode
           sudo apt-get install xvfb
           /usr/bin/xvfb-run npm run integration
+          cat /home/runner/.npm/_logs/*/*-debug.log
```

Comment:
I think this PR needs to:

- Ensure the exit code of the "Run integration tests" step is the exit code of npm run integration: currently I think the check will succeed if `npm run integration` fails but `cat` succeeds
- If the exit code of cat ends up mattering, test for the existence of these debug log files before we `cat` them so that `cat` returns a zero exit code